logstash.go

```diff
@@ -59,12 +38,33 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 	}, nil
 }
 
+// Returns a new slice containing all strings in the slice that satisfy the predicate f.
+func Filter(vs []string, f func(string) bool) []string {
+	vsf := make([]string, 0)
+	for _, v := range vs {
+		if f(v) {
+			vsf = append(vsf, v)
+		}
+	}
+	return vsf
+}
+
+// Get container tags configured with the environment variable LOGSTASH_TAGS
+func GetContainerTags(containerEnv []string) []string {
```

Comment:
This filter is used for filtering the environment variable which starts with "LOGSTASH_TAGS". You would prefer a simple loop on the environment variables and breaking when one is found correct?